### PR TITLE
Feature/personal info

### PR DESCRIPTION
**Pull request by YevheniiBieliaiev**
In this version:
1. Fixed routes - now we have for account page - /account, for subpages (personal info) - /personal-info
Example of full route - /account/personal-info
2. Fixed mq styles of components, added new styles
3. Fixed localization according to new rules
4. Fixed styles of icons according to last changes
5. Added validation for Personal Info section of form
6. Added validation for Password Section of form
7. Added action for reset button - change user data to storage data
8. Deleted all actions and code for default avatar , setted only field for it and upload button
only_current
same_emptyrepeat
notsame
space
cyr

### CODE REVIEW

File: packages/frontend/components/footer/links/policy.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const Policy: FC<PolicyProps> = ({ path }) => {
       <div css={styles.footerLinksRow}>
         <Icon
           cssExtend={styles.shield}
-          color="yellow"
+          color="YELLOW"
```
Comment (Nightlngale) [inline packages/frontend/components/footer/links/policy.tsx:19]: Let's use enum here

File: packages/frontend/components/primitives/dashboard-link/styles.ts [hunk 2]
```diff
@@ -22,8 +22,8 @@ export const icon = ({ spaces }: Theme) => css`
 
 export const img = () => css`
   display: block;
-  width: 22px;
-  height: 22px;
+  width: 18px;
+  height: 18px;
```
Comment (Nightlngale) [inline packages/frontend/components/primitives/dashboard-link/styles.ts:26]: dashboard-link component is not a primitive(not used across the App) + use button for it

File: packages/frontend/components/primitives/download-button/component.tsx [hunk 3]
```diff
@@ -1,24 +1,19 @@
...
+const InputFile = ({ id, multiple = false }: DownloadButtonProps, ref) => (
   <div css={styles.container}>
     <label css={styles.downloadWrapper} htmlFor={id}>
       <input
+        ref={ref}
         css={styles.download}
         id={id}
         type="file"
-        name={name}
+        accept=".jpg, .png, .jpeg"
```
Comment (Nightlngale) [inline packages/frontend/components/primitives/download-button/component.tsx:15]: Let's have separate component

File: packages/frontend/components/sub-pages/user-account/personal-info/styles.ts [hunk 4]
```diff
@@ -8,9 +8,12 @@ export const form = () => css`
   width: 100%;
 `;
 
-export const personalHeader = () => css`
+export const personalHeader = ({ mq }: Theme) => css`
   width: 100%;
-  margin-bottom: 7px;
+  margin-bottom: 20px;
```
Comment (Nightlngale) [inline packages/frontend/components/sub-pages/user-account/personal-info/styles.ts:13]: theme

**Review by Nightlngale — CHANGES_REQUESTED** [review]

File: packages/frontend/components/sub-pages/user-account/personal-info/styles.ts (hunk 4 above)
Comment (YevheniiBieliaiev) [reply packages/frontend/components/sub-pages/user-account/personal-info/styles.ts:13]: yup..))

**Review by Nightlngale — APPROVED** [review]

**Pull request merged**